shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/components/molecules/TopBarPresenter.kt

```diff
@@ -0,0 +1,39 @@
+package network.bisq.mobile.presentation.ui.components.molecules
+
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import network.bisq.mobile.domain.PlatformImage
+import network.bisq.mobile.domain.data.repository.UserRepository
+import network.bisq.mobile.presentation.BasePresenter
+import network.bisq.mobile.presentation.MainPresenter
+
+open class TopBarPresenter(
```

Comment:
@nostrbuddha had to add a TopBarPresenter in this PR hope that is ok, happy to help if you get conflicts.

Reply:
Sure. Rebased the UI code against this and making the UI PR now.